lib/galaxy/tool_util/linters/inputs.py

```diff
@@ -29,22 +29,31 @@ def lint_inputs(tool_xml, lint_ctx):
         if not is_valid_cheetah_placeholder(param_name):
             lint_ctx.warn("Param input [%s] is not a valid Cheetah placeholder.", param_name)
 
+        if param_type not in ["text", "integer", "float", "color", "boolean",
```

Comment:
This is already being verified through the schema. Try adding a nonsense parameter type and lint it, this will fail with
```
Applying linter tool_xsd... FAIL
.. ERROR: Invalid XML found in file: CreateInterval.xml. Errors [/var/folders/df/6xqpqpcd7h73b6jpx9t6cwhw0000gn/T/tmpwo5mb19x:7:0:ERROR:SCHEMASV:SCHEMAV_CVC_ENUMERATION_VALID: Element 'param', attribute 'type': [facet 'enumeration'] The value 'notype' is not an element of the set {'text', 'integer', 'float', 'color', 'boolean', 'genomebuild', 'library_data', 'select', 'data_column', 'hidden', 'hidden_data', 'baseurl', 'file', 'data', 'drill_down', 'group_tag', 'data_collection'}.]
```

Reply:
cool. undone this change. will fixup the commit.